accounting/accounting.go

```diff
@@ -97,6 +97,28 @@ func bytesAreZero(b []byte) bool {
 	return true
 }
 
+func (accounting *State) closeAccount(addr types.Address) {
+	if accounting.AlgoUpdates == nil {
+		accounting.AlgoUpdates = make(map[[32]byte]*idb.AlgoUpdate)
+	}
+
+	update, hasKey := accounting.AlgoUpdates[addr]
+
+	if !hasKey {
```

Comment:
this should also never happen because the account should have been touched earlier in the "pay" block of AddTransaction()